examples/radar/src/main.rs

```diff
@@ -0,0 +1,133 @@
+use std::rc::Rc;
+
+use yew::prelude::*;
+use yew_chart::{
+    axis::{Axis, Orientation, Scale},
+    linear_axis_scale::LinearScale,
+    series::{self, Labeller, Series, Type},
+};
+
+const WIDTH: f32 = 300.0;
+const HEIGHT: f32 = 300.0;
+const MARGIN: f32 = 50.0;
+const TICK_LENGTH: f32 = 2.0;
+
+#[function_component(App)]
+fn app() -> Html {
+    let labels = vec![
+        "Sales",
+        "Marketing",
+        "Development",
+        "Customer Support",
+        "Information Technology",
+        "Adminstration",
+    ];
+    let data_labels_60 = Rc::new(
+        series::to_radial(vec![60.0; 6])
+            .into_iter()
+            .zip(labels)
+            .map(|((x, y, _), label)| {
+                (
+                    x,
+                    y,
+                    Some(Rc::from(series::circle_text_label(label)) as Rc<dyn Labeller>),
+                )
+            })
+            .flat_map(|d| [d.clone(), (0.0, 0.0, None), d])
```

Comment:
This seems to write the label and the line to the origin twice.  In general, asking the use of the radial chart to draw the line to the origin is a bit obscure.   A new yew component <Radar ... /> would be ideal rather than forcing Series to draw this.

Reply:
Agreed. There's room for improvement on the UX there, but also, I want to apply the graph to a real-world app I'm writing before generalising the radar component any further.